src/ivoc/ocpicker.cpp

```diff
@@ -36,20 +36,17 @@ ButtonHandler::~ButtonHandler() {
     Resource::unref(rband_);
 }
 
-declarePtrList(HandlerList, ButtonHandler);
-implementPtrList(HandlerList, ButtonHandler);
-
 StandardPicker::StandardPicker() {
     ms_ = unknown;
     for (int i = 0; i < unknown; ++i) {
-        handlers_[i] = new HandlerList(1);
+        handlers_[i] = new std::vector<ButtonHandler*>();
+        handlers_[i]->reserve(1);
```

Comment:
Is there anything more to this line than it used to be `HandlerList(1)`, therefore, we `reserve(1)`? One question is, do we need to reserve at all?